NewsApp/app/src/main/java/com/thomasvondollen/newsapp/models/Category.kt

```diff
@@ -0,0 +1,5 @@
+package com.thomasvondollen.newsapp.models
+
+enum class Category() {
+    Business, Entertainment, General, Health, Science, Sports, Technology
+}
```

Comment:
Nice work on these enum classes and keeping them in separate files! one minor ajustment would be to make these enums all caps per the Kotlin coding convention.